dali/test/dali_test_single_op.h

```diff
@@ -42,7 +42,7 @@ const vector<string> jpeg_test_images = {
   image_folder + "/440.jpg",
   image_folder + "/444.jpg",
 //  image_folder + "/gray.jpg",   // Batched decoding has a bug in nvJPEG when both grayscale
-                                  //  and color images are decoded in the same batch
+                                //  and color images are decoded in the same batch
```

Comment:
@JanuszL is this still the case?

Reply:
After one more round of testing I will put back "gray.jpg" to this list. What I noticed, on my machine the eps's for  2 Color tests (Saturation/Hue) should be increased by 20-23%.